src/main/java/florizz/command/AddFlowerCommand.java

```diff
@@ -0,0 +1,59 @@
+package florizz.command;
+
+import florizz.core.FlorizzException;
+import florizz.core.FlowerDictionary;
+import florizz.core.Ui;
+import florizz.objects.Bouquet;
+import florizz.objects.Flower;
+
+import java.util.ArrayList;
+import java.util.concurrent.Flow;
+
+public class AddFlowerCommand extends Command{
+    private String flowerName;
+    private Integer quantity;
+    private String bouquetName;
+
+    public AddFlowerCommand(String flowerName, int quantity, String bouquetName) {
+        this.flowerName = flowerName;
+        this.quantity = quantity;
+        this.bouquetName = bouquetName;
+    }
+
+    @Override
+    public boolean execute(ArrayList<Bouquet> bouquetList, Ui ui) throws FlorizzException {
+        boolean doesBouquetExist = false;
+        Bouquet bouquetToAddFlower = new Bouquet();
+        for (int i = 0; !doesBouquetExist && i < bouquetList.size(); i++) {
+            if (bouquetList.get(i).getBouquetName().equals(this.bouquetName)) {
+                bouquetToAddFlower = bouquetList.get(i);
+                doesBouquetExist = true;
```

Comment:
why dont you throw the exception inside the for loop if there are any bouquets that match?